modules/react/button/stories/stories_Button.tsx

```diff
@@ -1,155 +1,119 @@
 /// <reference path="../../../../typings.d.ts" />
-/** @jsx jsx */
-import {jsx} from '@emotion/core';
+
+import React from 'react';
 import {storiesOf} from '@storybook/react';
 import withReadme from 'storybook-readme/with-readme';
 
 import {editIcon, playCircleIcon, activityStreamIcon} from '@workday/canvas-system-icons-web';
 
-import {Button} from '../index';
+import {Button} from '@workday/canvas-kit-react/button';
```

Comment:
All stories should have package name imports. Also this file was processed by codemods which requires imports from CK.